core/controllers/admin.py

```diff
@@ -671,21 +670,30 @@ def _load_dummy_new_structures_data(self) -> None:
+            fs_services.save_original_and_compressed_versions_of_image(
+            'thumbnail.svg', feconf.ENTITY_TYPE_TOPIC, topic_id_1, raw_image,
```

Comment:
This should be indented. See other simlar cases in this file.

Reply:
Done